### DISCUSSION TITLE
MQTT implementation

### DISCUSSION THREAD

**Discussion by lorepieri8** [django/channels#1675] [Q&A]
The docs introduction mentions the possibility of using MQTT with channels, but I cannot find any examples afterwards. 
Are there any reference implementations that I can look at?

**Comment by carltongibson** [comment]
I can't point to anything specific I'm afraid. [A search for 'mqtt django channels' seems to give some results](https://www.google.com/search?q=mqtt+django+channels).

**Comment by lbt** [comment]
I had the same issue and came up with:
https://pypi.org/project/chanmqttproxy/
https://github.com/lbt/channels-mqtt-proxy

I hope this uses the "proper" Channels approach.
